app/views/provider_management/external_users/disable_confirmation.html.haml

```diff
@@ -0,0 +1,10 @@
+= content_for :page_title, flush: true do
+  = t('.page_title', external_user: @external_user.name)
+
+%h1.govuk-heading-xl
+  = t('.page_title', external_user: @external_user.name)
+%h2.govuk-heading-m
+  = t('.disable_message', external_user: @external_user.name)
+.govuk-button-group
+  = govuk_link_button_warning(t('.disable_user'), disable_provider_management_provider_external_user_path(@provider, @external_user))
+  = govuk_link_to(t('.cancel_button'), provider_management_provider_external_user_path(@provider, @external_user))
```

Comment:
- you may wish to wrap this content in a two thirds column
- break up code to make it easier to read

Reply:
👍